### ISSUE TITLE
Question, not an Issue

### ISSUE THREAD

**Issue by Predacity** [bergant/xbrlus#7]
I am having trouble understanding how to set up the API key.  Where do I insert it, how do I reference it, etc?  You mention turning it into an .Renviron but then what? What should the file name be? Do we put the API with hyphens? Do we also put the variable and equal sign or just the api key. Please dumb it down and make this user friendly.

**Comment by bergant** [comment]
- The `.Renviron` is the file name. Please, note the dot at beginning and that there is *no file extension* at the end. You can use RStudio: create new file, select Save As and confirm the change of file extension.
- The .Renviron file should be created in your home directory. To check where your R home is, type normalizePath("~") int the console. In R Studio file viewer, home directory is named as "home" and marked with the house icon. 

![image](https://cloud.githubusercontent.com/assets/8250901/26541887/ee678d92-4457-11e7-8d3a-19b21d25aac4.png)

- If the .Renviron file already exists, just append the line in the existing file.

The file should contain a line formatted like this:

    XBRLUS_API_KEY=11791844-4446-11e7-8b15-15cb2f57b427

where 11791844-4446-11e7-8b15-15cb2f57b427 should be replaced by your key (it should be formatted as it is - with variable name and the hyphens).

I hope this made things a bit more clear.

**Comment by bergant** [comment]
Oh, I should also mention that this file is read at the beginning of every new R session. 
After you added the .Renviron file, restart RStudio or restart session (select Session / Restart R in the menu.)

**Comment by Predacity** [comment]
Works. Thanks
